netlify/functions/resources.ts

```diff
@@ -68,13 +68,45 @@ const resourcesHandler: Handler = async (event, context) => {
       body: tableToCsv(tableData),
     };
   } else if (resourceFilename === "findings.csv") {
-    const rawData = await readFile("./_data/findings/findings.csv", "utf8");
+    const findingsData = await getApiFindingsData();
+    let tableData: string[][] = [
+      [
+        "contest",
+        "handle",
+        "finding",
+        "risk",
+        "score",
+        "pie",
+        "split",
+        "slice",
+        "award",
+        "awardCoin",
+        "awardUSD",
+      ],
+      //contest,handle,finding,risk,score,pie,split,slice,award,awardCoin,awardUSD
+      ...findingsData.map((finding) => {
+        return [
+          "" + finding.contest,
+          finding.handle,
+          finding.finding,
+          finding.risk,
+          "" + finding.score,
```

Comment:
to match the current findings, this needs to generate a `""` string when `finding.score === null`

<img width="1026" alt="screenshot of two CSVs open in Numbers.app highlighting column value differences" src="https://user-images.githubusercontent.com/265902/232638924-9859685c-c15b-4bbd-be67-432d262bd617.png">

Reply:
```suggestion
          (finding.score === null) ? "" : ("" + finding.score),
```